### DISCUSSION TITLE
Bluetooth keyboard

### DISCUSSION THREAD

**Discussion by dpwe** [shorepine/tulipcc#28] [Ideas]
Seems like we have the hardware to support a bluetooth (wireless) keyboard.  That would be neat, wouldn't it?

**Comment by bwhitman** [comment]
It would be very neat! 
I cannot find on a quick Bing any existing Bluetooth keyboard host libraries for the esp32s3. I know it's possible.

**Reply by bwhitman** [reply]
https://github.com/turgu1/bt-keyboard

**Reply by dpwe** [reply]
Bing??? 🧌
